modules/issue_tracker/php/endpoints/attachment.class.inc

```diff
@@ -0,0 +1,259 @@
+<?php declare(strict_types=1);
+/**
+ * POST request methods.
+ *
+ * Used to handle issue_tracker attachment requests.
+ *
+ * PHP Version 7
+ *
+ * @category LORIS_Module
+ * @package  Login
+ * @author   Alizée Wickenheiser <alizee.wickenheiser@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+namespace LORIS\issue_tracker\endpoints;
+
+use \Psr\Http\Message\ServerRequestInterface;
+use \Psr\Http\Message\ResponseInterface;
+use \LORIS\Middleware\ETagCalculator;
+use \LORIS\api\Endpoint;
+
+/**
+ * The NDB_Page for the POST requests.
```

Comment:
I don't think this class extends NDB_Page

Reply:
@johnsaigle thanks removed.